django/http/multipartparser.py

```diff
@@ -207,8 +209,9 @@ def parse(self):
                             num_bytes_read > settings.DATA_UPLOAD_MAX_MEMORY_SIZE):
                         raise RequestDataTooBig('Request body exceeded settings.DATA_UPLOAD_MAX_MEMORY_SIZE.')
 
-                    self._post.appendlist(field_name,
-                                          force_text(data, encoding, errors='replace'))
+                    self._post.appendlist(
```

Comment:
Is `self._post.appendlist(field_name, force_text(data, encoding, errors='replace'))` too long?